e2e/protractor/protractor.excludes.json

```diff
@@ -6,14 +6,27 @@
   "C286252": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C284666": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C286269": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
-  "C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
-  "C308042": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
-  "C279186": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279219": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279221": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279220": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C325006": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
+  "C279191": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
+  "C279192": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
+  "C279193": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
```

Comment:
@swapnil-verma-gl why are you referring to https://alfresco.atlassian.net/browse/ACS-5183? That jira is in Verify state and I fixed already e2e which were failing as part of PR for that jira: 

https://github.com/Alfresco/alfresco-content-app/pull/3347/files

You should not mention that jira anymore. There are only two tests which were left and I was waiting for Michał and now I'm working on them, which are: 

"C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
"C308042": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632"

And these two should be assigned to ACS-5632

Reply:
Updated the protractor.excludes.json file. This information was probably overwritten during rebase.